### ISSUE TITLE
CKEditor 5 - unable to type in editor area with specific toolbar constellation

### ISSUE THREAD

**Issue by indigoxela** [backdrop/backdrop-issues#6363]
## Description of the bug

That's ... wow.

## Steps To Reproduce

1. On the latest dev remove the editor from Basic text format and save
2. Open the form on /admin/config/content/formats/filtered_html and add CKE5 back
3. Configure the toolbar to *not* contain the "heading" dropdown, save
4. Open a node form, e.g. /node/add/post
5. Try to type text (or paste text)

## Actual behavior

Nothing happens, you can't put in any text.

## Expected behavior

Well, a text editor should allow to type text.

## Additional information

It seems like paragraphs are prevented in this constellation, which prevents inserting *any* content.

**Comment by indigoxela** [comment]
Weird, if I switch to source mode and add a paragraph with content, everything starts working correctly.

BTW, when opening such a node form, the buttons for blockquote, list and alignment are disabled.
Nothing in the browser console. This seems to be "normal" behavior.

![no-text-input](https://github.com/backdrop/backdrop-issues/assets/842066/672f4384-22b9-4bed-9a06-373e85c41f41)

Weird, that adding a paragraph via source mode seems to unlock the editor.

**Comment by herbdool** [comment]
Maybe a solution would be similar to https://github.com/backdrop/backdrop-issues/issues/6359. Perhaps a basic paragraph dependency needs to be added if the heading plugin isn't loaded?

**Comment by stpaultim** [comment]
I've also confirmed this problem. I found this same bug while testing for https://github.com/backdrop/backdrop-issues/issues/6359

**Comment by indigoxela** [comment]
> Maybe a solution would be similar to https://github.com/backdrop/backdrop-issues/issues/6359.

@herbdool I don't think so. Because then the adding of an empty paragraph in source mode wouldn't fix it. It's probably yet another time one of the opinionated "features", this editor ships with. :wink:

**Comment by herbdool** [comment]
I've confirmed that https://github.com/backdrop/backdrop/pull/4635 fixes where the headings button is not enabled. But it still needs work since adding a heading button will now only show the paragraph option. I decided to publish the PR to get feedback.

**Comment by quicksketch** [comment]
Thanks @herbdool! I think this is a fine solution. We're doing the same thing to load other "always on" plugins such as AutoFormat and GeneralHtmlSupport.

**Comment by quicksketch** [comment]
Now with tests updated to include verifying the new plugin, this is back to RTBC.

**Comment by quicksketch** [comment]
Thanks @indigoxela, @herbdool, and @stpaultim! I merged https://github.com/backdrop/backdrop/pull/4635 into 1.x for 1.27.0.
